Sources/CloudAccessibility.swift

```diff
@@ -52,14 +52,4 @@ public enum CloudAccessibility: Int, CustomStringConvertible, Equatable {
     public var secAccessibilityAttribute: String {
         return accessibility.secAccessibilityAttribute
     }
-    
-    // MARK: Internal
-    
-    internal static func allValues() -> [CloudAccessibility] {
```

Comment:
Same comment here as above. Let's keep this property and add an appropriate `@available` tag.